websites/G/Gogoanime/dist/metadata.json

```diff
@@ -1,23 +1,26 @@
 {
   "author": {
-    "name": "PsychoPast",
+    "name": "Psycho",
     "id": "444779014888882177"
   },
+  "contributors": [{
+    "name": "Undo",
+    "id": "246660882803720193"
+  }],
```

Comment:
didnt undo want to be the author?

Reply:
Not really. I worked closely with them to implement the domain check and they're fine with it